src/_Problems_/binary-tree-to-binary-search-tree/binary-tree-to-binary-search-tree.test.js

```diff
@@ -0,0 +1,19 @@
+const { binaryTreeToBST, storeInorder } = require('.');
+const Node = require('./Node');
+
+describe('Binary tree to binary search tree', () => {
+  let tree;
+
+  describe('Create Binary Tree', () => {
+    tree = new Node(10);
+    tree.leftChild = new Node(30);
+    tree.leftChild.leftChild = new Node(20);
+    tree.rightChild = new Node(15);
+    tree.rightChild.rightChild = new Node(5);
```

Comment:
[Binary Tree](https://github.com/knaxus/problem-solving-javascript/blob/master/src/_DataStructures_/Trees/BinaryTree/index.js) implementation is already present.

Reply:
@k88manish, you can use this [Binary Tree using level order](https://github.com/knaxus/problem-solving-javascript/blob/master/src/_DataStructures_/Trees/BinaryTree) to create the tree here.